Tic-Tac-Toe-Python/main.py

```diff
@@ -0,0 +1,9 @@
+from TicTacToe import TicTacToeGame
+from UserPlayer import TicTacToeUserPlayer
+
+player_1 = TicTacToeUserPlayer('X', 'X')
+player_2 = TicTacToeUserPlayer('O', 'O')
+game1 = TicTacToeGame(player_1,player_2)
+
+game1.play()
```

Comment:
the logic for multiple game should be here

Reply:
This along with the previous comment means 
* shifting the logic for new game from TicTacToe.play() to main 
* creating a new object for every iteration of the loop 
* giving the option to user to specify the dimensions of the board

Is this correct?